cypress/integration/play_card_spec.js

```diff
@@ -0,0 +1,12 @@
+describe('Play card', () => {
+    beforeEach(() => {
+        cy.login('username');
+        cy.startGame();
+    });
+
+    describe('on word appearing', () => {
+        it('is able to play a card', () => {
+            // TODO: Write test
```

Comment:
When do you plan to write this test? Should this be in the PR?

Reply:
I'm blocked on this test until Sisi's multiple users stuff goes through